lib/syncthing/syncthing.go

```diff
@@ -119,12 +130,14 @@ func (a *App) startup() error {
 	})
 	a.mainService.ServeBackground()
 
+	a.mainService.Add(a.evLogger)
```

Comment:
I'm very skeptical of this. If New() takes an event logger it should assume it's already up and running, same as any other constructor taking an event logger.

Reply:
I see, that's indeed just consequent if we expose the event logging separately, which we do. I need to mentally drop (or much de-prioritize) wanting to keep the lib/syncthing interface as small and simple as possible :)